### PR TITLE
fix(users): Removing redundant variables

### PR DESCRIPTION
**Pull request by farajfarook**
Removing redundant variables defined and not used

### CODE REVIEW

File: modules/users/server/controllers/users/users.authentication.server.controller.js [hunk 1]
```diff
@@ -24,7 +24,6 @@ exports.signup = function (req, res) {
 
   // Init Variables
```
Comment (ilanbiala) [inline modules/users/server/controllers/users/users.authentication.server.controller.js]: Just combine this block and the add missing user fields into 1 block

**Comment by ilanbiala** [comment]
LGTM other than the one suggestion I made above.

File: modules/users/server/controllers/users/users.authentication.server.controller.js (hunk 1 above)
Comment (farajfarook) [inline modules/users/server/controllers/users/users.authentication.server.controller.js]: I'm sorry I didn't get you.

**Comment by codydaig** [comment]
After @ilanbiala's suggestions are made, LGTM.

File: modules/users/server/controllers/users/users.authentication.server.controller.js (hunk 1 above)
Comment (ilanbiala) [inline modules/users/server/controllers/users/users.authentication.server.controller.js]:
```
// Init user and add missing fields
var user = new User(req.body);
user.provider = 'local';
user.displayName = user.firstName + ' ' + user.lastName;
```

**Comment by farajfarook** [comment]
changes done as per @ilanbiala 's suggestion.

**Comment by codydaig** [comment]
LGTM

**Comment by lirantal** [comment]
LGTM

**Pull request merged**